bundle-workflow/src/manifests/test_manifest.py

```diff
@@ -40,7 +40,6 @@ class TestManifest(Manifest):
                     "integ-test": {
                         "type": "dict",
                         "schema": {
-                            "dependencies": {"type": "list"},
```

Comment:
You can remove this from the [example](https://github.com/opensearch-project/opensearch-build/pull/524/files#diff-b6591330ab14f0b34306021b8744f1e9c93e87a43e90cabee0024d9acb95a61aL19) above as well.